src/components/common/CountdownTimer.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { target } from "@/data/TargetDate";
+import { google} from "calendar-link";
+import { event } from "@/data/Calendar";
+
+
+export default function CountdownTimer() {
+  const [hackathonTime, setHackathonTime] = useState(false);
+  const [days, setDays] = useState(0);
+  const [hours, setHours] = useState(0);
+  const [minutes, setMinutes] = useState(0);
+  const [seconds, setSeconds] = useState(0);
+  
+  const googleUrl = google(event); 
```

Comment:
Small thing here, you could put this in a useMemo hook so it isn't recalculated every render as currently each time the component is re-rendered (each second) the googleUrl is being recalculated. useMemo allows you to memoize (cache) a result.